{"uuid":"8001762b-1a2f-4efe-93b9-d40c2080e3bf","historyId":"7dea2f727c0e6b190e9d192625ebe3ff","fullName":"webTestCase.verifyUserSignInDataDriven.vendorSignInDataDriven","labels":[{"name":"package","value":"webTestCase.verifyUserSignInDataDriven"},{"name":"testClass","value":"webTestCase.verifyUserSignInDataDriven"},{"name":"testMethod","value":"vendorSignInDataDriven"},{"name":"parentSuite","value":"All Test Cases"},{"name":"suite","value":"Vendor Login"},{"name":"subSuite","value":"webTestCase.verifyUserSignInDataDriven"},{"name":"host","value":"DESKTOP-Q1U6PAM"},{"name":"thread","value":"3168@DESKTOP-Q1U6PAM.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"vendorSignInDataDriven","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Element not exist. By.xpath: //strong[text()=\"These credentials do not match our records.\"]","trace":"java.lang.AssertionError: Element not exist. By.xpath: //strong[text()=\"These credentials do not match our records.\"]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat keywords.WebUI.waitForElementPresent(WebUI.java:2930)\r\n\tat keywords.WebUI.waitForElementVisible(WebUI.java:2839)\r\n\tat mawad.web_pages.SignInPage.signIn(SignInPage.java:49)\r\n\tat webTestCase.verifyUserSignInDataDriven.vendorSignInDataDriven(verifyUserSignInDataDriven.java:21)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg0","value":"{ACTUAL_RESULT=, EXPECTED_ERROR=, EXPECTED_URL=, TESTCASENAME=login fail with wronge username and correct password, USERNAME=worngeusername, TESTCASE_ID=LI_06, EXPECTED_TITLE=fail, PASSWORD=Qwerty003!}"}],"start":1719908837567,"stop":1719908863891}